pages/linux/rpm-ostree.md

```diff
@@ -0,0 +1,29 @@
+# rpm-ostree
+
+> A hybrid image/package system.
+> Manage ostree deployments, package layers, filesystem overlays, and boot configuration.
+> More information: <https://coreos.github.io/rpm-ostree/administrator-handbook/>.
+
+- Show rpm-ostree deployments in the order they'll appear in the bootloader:
+
+`rpm-ostree status`
+
+- Show package updates that will be performed on the next rpm-ostree upgrade:
+
+`rpm-ostree upgrade --preview`
+
+- Prepare a new ostree deployment with upgraded packages and reboot into the new deployment:
+
+`rpm-ostree upgrade --reboot`
+
+- Roll back and reboot into the previous ostree deployment:
+
+`rpm-ostree rollback --reboot`
+
+- Install a package into a new ostree deployment and apply that deployment to the live deployment:
```

Comment:
```suggestion
- Install a package into a new ostree deployment and apply that deployment to the current live deployment:
```

Reply:
Not sure if it's still correct, but what about this
```suggestion
- Install a package into a new ostree deployment and apply it to the current live deployment:
```

Reply:
I'll stew on this for a bit, `rpm-ostree` is a pretty strange (and somewhat complex) beast and I'm having trouble coming up with language that's precise/accurate/approachable. This language is more clear but there's some complex behavior happening with filesystem overlays, and I _think_ there's some unique behavior with this on reboots that needs to be captured.

If I can't figure out a good way to explain this in the next couple of hours then I'll drop this example and open up a new PR for `rpm-ostree install --apply-live` so this PR doesn't get stuck on my indecision. 😁

Reply:
> I'll stew on this for a bit, `rpm-ostree` is a pretty strange (and somewhat complex) beast and I'm having trouble coming up with language that's precise/accurate/approachable. This language is more clear but there's some complex behavior happening with filesystem overlays, and I _think_ there's some unique behavior with this on reboots that needs to be captured.
> 
> If I can't figure out a good way to explain this in the next couple of hours then I'll drop this example and open up a new PR for `rpm-ostree install --apply-live` so this PR doesn't get stuck on my indecision. 😁

Hey, no need to remove the example from this PR, I will try to come up with an example

Reply:
So, We can remove this command from this page currently as mentioned in [comment](https://github.com/tldr-pages/tldr/pull/8277/files/223d1b30ef57429027481cc9c70283758971909c#r936174001).

Reply:
Yeah, let's avoid the complexity and drop it. `rpm-ostree install -A {{package}}` seems too risky to have as the first 8 commands we should be sharing with users.